cmd/utils/executor.go

```diff
@@ -58,8 +58,10 @@ func NewExecutor(output string) *Executor {
 }
 
 func IsSupportForTTY() bool {
-	_, err := unix.IoctlGetWinsize(int(os.Stdout.Fd()), unix.TIOCGWINSZ)
-	return err == nil
+	if runtime.GOOS != "windows" {
```

Comment:
Isn't this checked in the file compiled only for windows? Why would we need to add also this check here?